va/va.h

```diff
@@ -748,6 +748,13 @@ typedef enum
      * support for QP info for buffer #VAEncQpBuffer.
      */
     VAConfigAttribQPBlockSize            = 37,
+    /**
+     * \brief frame type attribute. Read-only.
+     * this attribute conveys the frame type for specific config
+     * the value could be  VA_FRAME_TYPE_XXXX. for VPX, interframe also can be  VAA_FRAME_TYPE_P
+     * by default I frame is supported, no need to report it.
```

Comment:
For backward compatibility it might be a good idea to write that if this attribute is not present then frame type support can be deduced from `VAConfigAttribEncMaxRefFrames` instead.